OloEngine/src/OloEngine/Scripting/Lua/LuaScriptEngine.cpp

```
@@ -2,6 +2,9 @@
 #include "LuaScriptEngine.h"
 #include "LuaScriptGlue.h"
 
+#include "OloEngine/Scene/Scene.h"
```

Cppcheck (2.20.0)

[information] Include file

(missingInclude)